### ISSUE TITLE
Composer?

### ISSUE THREAD

**Issue by emberdart** [etiennetremel/PHP-Find-Date-in-String#7]
Can this be added to composer so people can use it in their projects and keep up to date with modifications  and attributions to your code?
Cheers

**Comment by etiennetremel** [comment]
Yes that would definitely be nice! I kind of drop doing any PHP development for the last few years so my PHP skills are bit rusty. If I have time I will jump on it, meanwhile PRs are more than welcome.
Other task to take in consideration: unit testing!
